src/geometry.rs

```diff
@@ -393,7 +439,7 @@ pub fn cube_smooth_same_len(position: [f32; 3], scale: f32) -> Geometry {
     Geometry::from_triangle_faces_with_vertices_and_normals(faces, vertex_positions, vertex_normals)
 }
 
-pub fn cube_sharp_same_len(position: [f32; 3], scale: f32) -> Geometry {
```

Comment:
I suggest removing this function because it doesn't generate a proper watertight mesh. it's being used in one of the tests but that can be replaced by some other function

Reply:
While I see that the mesh it generates probably makes no sense for geometry modeling, I would prefer to keep if for a while longer. We can add a FIXME to remove, but I want to double check the generated renderer geometry from `cube_sharp_var_len()` is equivalent to this one before we remove.

Reply:
it generates a broken geometry. please, check if the new one is identical (as far as i'm concerned, it is) and let's please remove it so that we don't use it for further tests.

Reply:
after a debate with @yanchith marked as `#[deprecated]`